doc/index.rst

```diff
@@ -20,13 +21,13 @@ your BinderHub deployment.
 To get started creating your own BinderHub, start with :doc:`create-cloud-resources`.
 
 Extending JupyterHub
---------------------
+====================
```

Comment:
Not directly related to this PR but the "Extending JupyterHub" section feels a bit out of place here.

Reply:
Ah that's a good point - do you have a suggestion for a different wording / title here? I'm happy to add it into this PR

Reply:
Honestly, I think that I would remove this section completely and link to the z2jh in the create-cloud-resources doc. :shrug: It feels out of place here and dilutes the emphasis on BinderHub.

Reply:
ok, see the latest push. I removed the header and replaced it with a `.. note` directive on the index page and the cloud resources page. Does that work?